### ISSUE TITLE
Bug: (fill in): eslint.config.js

### ISSUE THREAD

**Issue by Jainil2** [eslint/eslint#19388]
### Environment

Node version:23.6.1
npm version:10.9.2
Local ESLint version:9.19.0
Global ESLint version:-
Operating System:Windows 11

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```
<!-- Paste your configuration here -->
```
import globals from "globals";
import pluginJs from "@eslint/js";

/** @type {import('eslint').Linter.Config[]} */
export default [
  {languageOptions: { globals: globals.browser }},
  pluginJs.configs.recommended,
];
</details>

```js
<!-- Paste your code here -->
```

### What did you expect to happen?

 {languageOptions: { globals: globals.browser }}, here globals should globals.node

### What actually happened?

 {languageOptions: { globals: globals.browser }}, configuration take browser even we select node inside configuration step

### Link to Minimal Reproducible Example

https://github.com/Jainil2

### Participation

- [x] I am willing to submit a pull request for this issue.

### Additional comments

it affect process module in nodejs and give error when we run eslint.

**Comment by Tanujkanti4441** [comment]
Hi @Jainil2, thanks for the issue, but it is working as intended.


The field `Where does your code run?` is a multi selector and to choose the environment you want you have to press `space` after going to that option.

![Image](https://github.com/user-attachments/assets/9bd848a8-124d-4b32-a224-d311a816735b)

As you can see here by the green check the `browser` option is selected and if we click enter we will get `globals.browser` in config, to select `node` we can press space.

![Image](https://github.com/user-attachments/assets/20163431-9448-420b-a787-b7fe0f9c887a)

Is that solves your issue?

**Comment by Tanujkanti4441** [comment]
Closing! since it's working as intended.
